docs/fundamentals/flow/monitoring-flow.md

```diff
@@ -91,6 +91,51 @@ On the other hand, If you want to only enable the monitoring on a given Executor
 Flow().add(...).add(uses=MyExecutor, monitoring=True)
 ```
 
+### Enable monitoring with replicas and shards
+
+When using replicas or shards (see this how-to scale-out) all of the duplicate Executor will expose their own metrics.
+
+When deploying a Flow locally (i.e without Kubernetes or Docker Compose) all the duplicate will use a random `port_monitoring`
+expect one that will use the `port_monitoring` that you precise. You can as well pass a list of ports to be used by the different replicas 
+for the monitoring by passing a string of several ports separated by a comma.
```

Comment:
Could we rephrase this?
I would first explain how to pass a list using the comma, then give the examples.
Only below that i would describe what happens if only one port is given, as that seems like a special case to me (and there i would first say that one replica gets the specified port, and the other ones are random, instead of the other way around. I think that would be clearer.)

Reply:
Sure